src/components/JourneyDisplayComponents/StatusDisplay.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { HiCheckCircle, HiExclamationCircle, HiXCircle } from 'react-icons/hi';
+import { MdGroupOff } from 'react-icons/md';
+
+interface StatusArgs {
+    status: string;
+}
+
+const StatusDisplay: React.FC<StatusArgs> = ({ status }) => {
+
+    return (
+        <span className = 'status-container' 
+            style = { { backgroundColor: status!=='normal'? '#F4442E' : '#187795' } }>
```

Comment:
Again could the styling be linked to a class name instead,
Would make adding a new option easier (e.g. flying train) as all the styles for these bits would be in 1 place?

Reply:
used class names for this